js/core/src/flow.ts

```diff
@@ -145,9 +145,7 @@ export type FlowFn<
   /** Input to the flow. */
   input: z.infer<I>,
   /** Callback for streaming functions only. */
-  streamingCallback?: S extends z.ZodVoid
-    ? undefined
-    : StreamingCallback<z.infer<S>>
+  streamingCallback?: StreamingCallback<z.infer<S>>
```

Comment:
What changed for this to no longer be needed?

Reply:
Schema is technically optional...